### ISSUE TITLE
how change project, post icon image? 

### ISSUE THREAD

**Issue by Vallista** [TaylanTatli/Halve#41]
I looked a lot to change the icon, but there was no image.
where is the image?

**Comment by libeclipse** [comment]
There's an images folder.

**Comment by Vallista** [comment]
I can not find it there. And not in the images folder of the project. :(

**Comment by libeclipse** [comment]
https://github.com/TaylanTatli/Halve/blob/master/images/halve.png
https://github.com/TaylanTatli/Halve/blob/master/images/halve-small.png

**Comment by Vallista** [comment]
no, What I'm talking about is not the project, post icon on the top right. Not a profile picture

**Comment by Vallista** [comment]
this is source code control image.

**Comment by tanto259** [comment]
If I am not mistaken, those logo are generated through CSS. Try searching `/assets/_scss/site.scss`, heres the link for the source code: https://git.io/v9Y6f
